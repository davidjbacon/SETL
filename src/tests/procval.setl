s := {ROUTINE c, ROUTINE b, ROUTINE a};
WHILE #s > 0 LOOP
  p FROM s;
  CALL(p);
END;

PROC a; PRINT('a');b;c; END;
PROC b; PRINT('b');c; END;
PROC c; PRINT('c'); END;
