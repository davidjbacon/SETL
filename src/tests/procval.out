a
b
c
c
b
c
c
